core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -15,6 +15,13 @@
 
     <div ng-repeat="question in data.questionList.getQuestions() track by $index">
       <anchor-header header-id="question.getId()" header-text="Question <[$index +1]>"></anchor-header>
+      <anchor-header header-id="getSubfieldId(question,'Multiple Choice')"
+                     header-text="<[getQuestionTypeLabel(question.getInteractionId())]> &#x25BC;" ng-click="enableChooseQuestion(); setModalVisible()">
+      </anchor-header>
+
+      <div ng-show="chooseQuestionType && $parent.modalWrapperVisible" class="question-type-wrapper">
```

Comment:
Try not to use $parent, that's an antipattern. Not sure I can make further suggestions here since I don't understand what modalWrapperVisible actually is for.